FAILED: pgo
ld: error: undefined symbol: key_binding_unref_term
>>> referenced by ld-temp.o
>>>               lto.tmp:(term_destroy)

ld: error: undefined symbol: key_binding_new_for_term
>>> referenced by ld-temp.o
>>>               lto.tmp:(term_init)
cc: error: linker command failed with exit code 1 (use -v to see invocation)

--- pgo/pgo.c.orig	2022-04-22 15:14:41 UTC
+++ pgo/pgo.c
@@ -193,6 +193,17 @@ key_binding_for(
     return &kbd;
 }
 
+void
+key_binding_new_for_term(struct key_binding_manager *mgr,
+                         const struct terminal *term)
+{
+}
+void
+key_binding_unref_term(struct key_binding_manager *mgr,
+                       const struct terminal *term)
+{
+}
+
 int
 main(int argc, const char *const *argv)
 {
